lib/tasks/rake_config.rb

```diff
@@ -6,16 +6,30 @@ def context
 
     attr_writer :context
 
-    def config
+    def config(is_cc_local_worker: false)
       secrets_hash = {}
       # TODO: require secrets fetcher?
       secrets_hash = VCAP::CloudController::SecretsFetcher.fetch_secrets_from_file(secrets_file) unless secrets_file.nil?
 
-      VCAP::CloudController::Config.load_from_file(config_file, context:, secrets_hash:)
+      api_config = VCAP::CloudController::Config.load_from_file(config_file, context:, secrets_hash:)
+
+      if is_cc_local_worker
+        local_worker_config_hash = VCAP::CloudController::Config.read_file(cc_local_worker_config_file)
```

Comment:
My proposal to minimize duplicate code execution is to:
1. read the ccng config file (`Config.read_file`)
2. if `context` is `:api` and local_worker_override file exists
  a. read the local_worker_override file (`Config.read_file`)
  b. merge config hashes
3. load config (`Config.load_from_hash`)